src/theory/output_channel.h

```diff
@@ -10,88 +10,62 @@
  * directory for licensing information.
  * ****************************************************************************
  *
- * The theory output channel interface.
+ * The theory engine output channel.
  */
 
 #include "cvc5_private.h"
 
-#ifndef CVC5__THEORY__OUTPUT_CHANNEL_H
-#define CVC5__THEORY__OUTPUT_CHANNEL_H
+#ifndef CVC5__THEORY__ENGINE_OUTPUT_CHANNEL_H
+#define CVC5__THEORY__ENGINE_OUTPUT_CHANNEL_H
```

Comment:
Since the header file is still `output_channel.h` the guard should be `CVC5__THEORY__OUTPUT_CHANNEL_H`.